app/src/main/java/com/example/admin/personallibrarycatalogue/MainActivity.java

```diff
@@ -0,0 +1,62 @@
+package com.example.admin.personallibrarycatalogue;
+
+import android.app.FragmentManager;
+import android.app.FragmentTransaction;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.support.v7.app.ActionBarActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+
+
+public class MainActivity extends ActionBarActivity {
+
+    SharedPreferences preferences = null;
+    private static final String APP_NAME = "com.example.admin.personallibrarycatalogue";
+    private static final String FIRSTRUN = "firstrun";
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        preferences = getSharedPreferences(APP_NAME, MODE_PRIVATE);
```

Comment:
Guess we should get rid of SP. You should rely only on what DB contains at the moment.